pkgs/by-name/pl/pluginval/package.nix

```
@@ -0,0 +1,92 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  alsa-lib,
+  gtk3,
+  freetype,
+  fontconfig,
+  xorg,
+  curl,
+  webkitgtk_4_1,
+  libsysprof-capture,
+  pcre2,
+  util-linux,
+  libselinux,
+  libsepol,
+  libthai,
+  libdatrie,
+  libXdmcp,
+  libdeflate,
+  lerc,
+  xz,
+  libwebp,
+  libxkbcommon,
+  libepoxy,
+  libXtst,
+  sqlite,
+  ladspaH,
+}:
+stdenv.mkDerivation rec {
+  pname = "pluginval";
+  version = "1.0.4";
+  src = fetchFromGitHub {
+    owner = "Tracktion";
+    repo = "pluginval";
+    rev = "v${version}";
+    sha256 = "sha256-j4Lb3pcw0931o63OvTTaIm2UzvYDIjmnaCXGvKB4gwM=";
```

Use `hash` and refill a new one here.

```suggestion
    hash = "...";
```